src/SprykerEco/Yves/Payone/Form/DataProvider/BancontactOnlineTransferDataProvider.php

```diff
@@ -36,14 +37,16 @@ public function __construct(PayoneConfig $config)
      */
     public function getData(AbstractTransfer $quoteTransfer): AbstractTransfer
     {
-        /** @var \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer */
-        if ($quoteTransfer->getPayment() !== null) {
+        if ($quoteTransfer instanceof QuoteTransfer && $quoteTransfer->getPayment() !== null) {
```

Comment:
```suggestion
        if ($quoteTransfer->getPayment() !== null) {
```

Reply:
If we leave it like this, PHPStan screams that the getPayment() method is not in AbstractTransfer, because the object comes here QuoteTransfer, in which this method is. If you put an inline comment specifying what kind of object we have here, then the sniffer will scream in the code-style.